Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: build
 build:
 	cargo build --release
+	cargo run --release -- docs --output cli-docs.md
```

Comment:
On second thought, given that we do not do `make build` on the build/release job, maybe it is a good idea to separate this into a different `build-docs` that the `build` target depends on or so, so we can call `make build-docs` later on the build/release job to ship them on the GH release.